docs/eventlisteners.md

```diff
@@ -9,15 +9,16 @@ with.
 When an `EventListener` is successfully created, a service is created that
 references a listener pod. This listener pod accepts the incoming events and
 does what has been specified in the corresponding
-`TriggerBinding`s/`TriggerTemplate`s.
-
+`TriggerBinding`s/`TriggerTemplate`s. The service created is a `ClusterIP` service 
```

Comment:
Spoke with you on this, will be updated if/when the service type configuration PR is merged.

Reply:
ack, adding it once we have #175 merged